package.json

```diff
@@ -16,14 +16,14 @@
     "src": "./src"
   },
   "scripts": {
-    "start": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run build:vendor && yarn run start:dev",
-    "start:cloud": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run build:vendor && yarn run start:dev-cloud",
```

Comment:
Ah, wait. May have answered my own question. Webpack docs suggest that the `optimization.splitChunks.cacheGroups` will bundle vendors separately as the default. But I believe we only use splitChunks in the prod build, not the dev build.

Reply:
>It looks like the `webpack.vendor.ts` is never consumed anymore. Is this correct? So we are undoing the separate bundling as per [this example in the docs](https://webpack.js.org/concepts/entry-points/#separate-app-and-vendor-entries)?

@wiedld This is correct. I had trouble getting the vendor configuration to run without erroring out, so I tried removing it to see if it worked. From what I read, vendor bundles are a dev-level optimization, since on a production build, there isn't any reloading happening, so optimizing our code from our vendors' code isn't as important.

I'll look into using `splitChunks` to achieve something similar, thanks.

Reply:
thanks for removing the vendor config!